src/Maker/MakeTest.php

```diff
@@ -77,10 +78,11 @@ public function configureCommand(Command $command, InputConfiguration $inputConf
         }
 
         $command
-            ->addArgument('name', InputArgument::OPTIONAL, 'The name of the test class (e.g. <fg=yellow>BlogPostTest</>)')
             ->addArgument('type', InputArgument::OPTIONAL, 'The type of test: '.implode(', ', $typesDesc))
+            ->addArgument('name', InputArgument::OPTIONAL, 'The name of the test class (e.g. <fg=yellow>BlogPostTest</>)')
```

Comment:
Once I played with this, I decided to reverse the arguments. It occurred to me that knowing the "type" as early as possible could help us. For example, I don't do it in this PR, but we could suggest different example filenames on the next question based on the type.